### ISSUE TITLE
Internationalisation de l'outil

### ISSUE THREAD

**Issue by nondidju** [progracqteur/WikiPedale#49]
S'il s'agit pour l'instant d'un projet très local et purement francophone, nous devons nous préparer à une récupération et utilisation de l'outil par d'autres mouvements à l'étranger. Une communauté de développeurs pourrait alors s'emparer de cet outil et l'améliorer.

Pour ça, nous devons dès maintenant adopter des comportements simples :
- Utiliser au maximum l'anglais dans les noms de balises, commentaires,...
- Tous les textes destinés à être affichés doivent être enregistrés dans le fichier FOSUserBundle.fr.yml afin de faciliter le travail des traducteurs
- Tout ce que nous écrivons doit être explicite et compréhensible par d'autres, commentons tout ce qui peut l'être dans le code.

**Comment by julienfastre** [comment]
Oui, c'est une attention que l'on devrait avoir. Je pense qu'il faut que je commente plus mon travail. 

Pour le fichier, il doit s'agir plutot du fichier WikipedaleUserBundle.fr.yml (mais il est à créer, et Benoit tu sembles avoir lu la doc pour savoir où le placer). FOSUserBundle est un fichier d'un autre bundle importé (celui qui gère les inscriptions, login, etc.) . Si eux mettent à jour ce fichier, toutes nos modifications seront écrasées !

**Comment by marcu** [comment]
FOSUserBundle ne marche pas chez moi. J'ai fait un php "php bin/vendors install" et nettoyage du cache mais j'ai toujours "layout.login" la la place de "login"

**Comment by julienfastre** [comment]
@marcu j'ai ajouté les traductions du message: ça fonctionne chez moi, après un `php app/console cache:clear` . 

@nondidju : 
- Je pense qu'il faut créer un fichier de message par type de message (un "domaine" dans Symfony http://symfony.com/doc/2.0/book/translation.html#using-message-domains ), ça me semble plus clair. J'avais créé le domaine "changes" pour caractériser tous les changements faits sur un emplacement; à toi de voir si tu as du courage de déplacer les messages vers un autre fichier et le domaine dans les fonctions... :-)
- Dans toute la syntaxe du projet, j'ai appelé "place" les emplacements, et j'ai vu que tu appelais "black_points" les emplacements. A mon avis, ca serait plus clair qu'on utilise un seul vocabulaire (surtout que "black_point" en anglais, je ne sais pas si ça existe ni si ça veut dire quelque chose :-) )

Mes modifications ici : e6ec9c53a26537785571dfe7e699441c5b0c1a53

**Comment by marcu** [comment]
ben ça marche pas.

**Comment by julienfastre** [comment]
J'ai mis à jour la page d'installation pour ajouter la petite manoeuvre qui facilite la vie... https://github.com/progracqteur/WikiPedale/wiki/Installer-wikipedale-et-symfony , sous le point trois: compléter la variable locale dans parameters.ini .
